src/App.tsx

```diff
@@ -1,8 +1,17 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import logo from './logo.svg';
 import './App.css';
+import axios from 'axios';
 
 function App() {
+
+  const [data, setData] = useState("");
+  useEffect(() => {
+    axios.get(`${process.env.REACT_APP_API_URL}`).then((res) => {
+      setData(res.data);
+    })
+  }, [])
```

Comment:
- Preferowalnie chcialibyśmy wydzielić geta do osobnej funkcji, np. getData, teraz nazwa nie jest ważna bo i tak ten endpoint zniknie w tej postaci, ważne żeby już teraz używać dobrych praktyk
- Rozważyłbym wykorzystanie [fetch API](https://jsonplaceholder.typicode.com/guide/), tylko dlatego że jest wbudowane i przez to jeśli nie ma wyraźnych przeciwskazań, powinno być wykorzystane